src/devhub/entity/addon/github/kanban_board.jsx

```diff
@@ -127,12 +127,17 @@ const GithubKanbanBoard = ({
           const tickets = state.ticketsByColumn[column.id] ?? [];
 
           return (
-            <div className="col-4" key={`column-${column.id}-view`}>
+            <div
+              className="col-3"
+              style={{ minWidth: "300px" }}
+              key={`column-${column.id}-view`}
+            >
               <div className="card rounded-4">
                 <div
+                  style={{ height: "75vh", overflow: "scroll" }}
                   className={[
                     "card-body d-flex flex-column gap-3 p-2",
-                    "border border-2 border-secondary rounded-4",
+                    "border border-1 border-secondary rounded-4",
```

Comment:
I think in order to make it a bit more like github, we can also remove `border-secondary` here. Same goes for the cards.